BPOR/BPOR.Rms/Controllers/CampaignController.cs

```diff
@@ -92,27 +108,48 @@ public async Task<IActionResult> SendEmail(SetupCampaignViewModel model, Cancell
                 {
                     if (string.IsNullOrWhiteSpace(recipient))
                     {
-                        logger.LogWarning("Empty notification email address for study ({studyId}) '{studyName}', email campaign ({emailCampaignId}).", model.StudyId, model.StudyName, emailCampaign.Id);
+                        logger.LogWarning("Empty notification email address for study ({studyId}) '{studyName}', campaign ({campaignId}).", model.StudyId, model.StudyName, campaign.Id);
 
                         continue;
                     }
 
-                    await transactionalEmailService.SendAsync(recipient, "email-rms-campaign-sent", new { numberOfVolunteers = model.TotalVolunteers, studyName = studyInfo.StudyName }, cancellationToken);
+                    object sendParams = new { };
+
+                    if (string.IsNullOrWhiteSpace(model.StudyName))
+                    {
+                        sendParams = (new { numberOfVolunteers = model.TotalVolunteers, letterTemplateFilename = selectedTemplate });
+                    }
+                    sendParams = (new { numberOfVolunteers = model.TotalVolunteers, studyName = studyInfo.StudyName });
```

Comment:
`sendParams` reassigned immediately.
we can determine the params by `campaign.TypeId` below